shared/js/background/chrome-events.es6.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * NOTE: this needs to be the first listener that's added
+ *
+ * on FF, we might actually miss the onInstalled event
+ * if we do too much before adding it
+ */
+const ATB = require('./atb.es6')
+
+chrome.runtime.onInstalled.addListener(function(details) {
+    // only run the following section on install and on update
+    if (details.reason.match(/install|update/)) {
+        ATB.updateATBValues();
+    }
+
+    if (details.reason.match(/install/)) {
+        ATB.openPostInstallPage()
+    }
+})
+
+/**
+ * REQUESTS
+ */
+
+const constants = require('../../data/constants')
+const redirect = require('./redirect.es6')
+const tabManager = require('./tab-manager.es6')
+
+chrome.webRequest.onBeforeRequest.addListener(
+    redirect.handleRequest,
```

Comment:
If `background.js` is the entry point, and this is where the events are added, I figured the onBeforeRequest handler needs to live inside a separate module.

Open to better naming suggestions :)